### DISCUSSION TITLE
Why does Discussions on GitHub exist as well as the Discord server having a forum style help channel?

### DISCUSSION THREAD

**Discussion by kleenkanteen** [medusajs/medusa#4234] [Q&A]
For Q&A they both serve the same purpose. The drawback is that one would have to search both places instead of everything being in one place. I think all Q&A should be centralized in Discord. What do you guys think?

**Comment by pevey** [comment]
Discussions here on GitHub are often used in a different way than on Discord.  The forum-style Help channel is just that, users asking other users for help.  Sometimes those can lead to things being added as a GitHub Issue or Discussion, but most often not.  Mostly those threads are just needing another set of eyes to see where some piece of code or configuration has gone wrong.  Discussions are intended to relate more to feature discussions and the overall direction of the project, not specific help requests.

**Reply by kleenkanteen** [reply]
Thing is there is a Q&A category here and people are using it, it's not just overall direction of the project:

![image](https://github.com/medusajs/medusa/assets/67878058/cfb5b51d-12c7-400f-b792-c5f72bc71ee8)

The Discord channel has a help channel. A lot of good info is there. It unblocked me when I searched it regarding a error railway had needing a developer plan if you want to host the admin dashboard there.

If the Q&A category here was removed I think that'd be better as Discord is much easier to talk back and forth which when getting help is constant.

@shahednasser Thoughts?

**Comment by agustif** [comment]
This could be a potential solution to exposing/indexing Discord to the open internet https://news.ycombinator.com/item?id=36383773

**Reply by kleenkanteen** [reply]
Interesting idea, this could allow both discord and github Q&A to exist as both are accessible with a google search. And people can use whatever they like best (even though github is a slower expereince :p).

**Reply by agustif** [reply]
Yes, it's pretty new, the only similar tools I know I have seen on the wild where for Slack instead, using linen https://www.linen.dev/

Leaving a note here in case it applies in the future here or on other projects reading this thread.

In any case, it seems smart to try and index everything, but also communities die by perfect FAQ/indices as in newcomers are really not encouraged to ask anything if everything is answered.

Maybe going full circle would be having an LLM based Agent on the chat based communities like discord, that can actually answer with factual data from the past if someone asks something already answered on those... heh

Just saw this FOSS Discord alternative on the homepage of HN today too, I know nobody wants to migrate, but cool to see open options appear, at least for open source software might make more sense into the future https://revolt.chat/
